src/Options/ParseOptions.hpp

```diff
@@ -388,30 +404,60 @@ void Options<OptionList>::parse(const YAML::Node& node) {
   }
 }
 
-template <typename OptionList>
-template <typename T, typename Metavariables>
-typename T::type Options<OptionList>::get() const {
-  static_assert(
-      not cpp17::is_same_v<tmpl::index_of<opts_t, T>, tmpl::no_such_type_>,
-      "Could not find requested option in the list of options provided. Did "
-      "you forget to add the option struct to the OptionList?");
-  const std::string label = Options_detail::name<T>();
-
-  validate_default<T>();
-  if (0 == parsed_options_.count(label)) {
-    return get_default<T>();
+namespace Options_detail {
+template <typename T, typename Metavariables, typename Subgroup>
```

Comment:
maybe here too we should do `T` -> `Tag` and move away from my poor decisions :)